build.sbt

```diff
@@ -845,7 +848,8 @@ inScope(Global)(
     credentials ++= (for {
       username <- sys.env.get("SONATYPE_USERNAME")
       password <- sys.env.get("SONATYPE_PASSWORD")
-    } yield Credentials("Sonatype Nexus Repository Manager", "oss.sonatype.org", username, password)).toSeq,
+    } yield
+      Credentials("Sonatype Nexus Repository Manager", "oss.sonatype.org", username, password)).toSeq,
     PgpKeys.pgpPassphrase := sys.env.get("PGP_PASSPHRASE").map(_.toCharArray())
   )
 )
```

Comment:
Looks like build.sbt has been inadvertently reformatted.

Reply:
Yes, that may have been caused by vscode reformatting on save 😅 I've added build.sbt to .scalafmt.conf